mail.go

```diff
@@ -132,3 +132,27 @@ func (c *Client) SendBatchHTMLEmail(req SendBatchHTMLEmailReq) (res *SendBatchHT
 	return &response, nil
 }
 
+func (c *Client) FileCacheUploadAPI(req FileCacheUploadAPIReq) (res *FileCacheUploadAPIRes, err error) {
+	url := "https://api.zeptomail.com/v1.1/files?name="
```

Comment:
@evans-sudo Kindly update the URL to use the BaseURL from the client and properly fix the other parts of the URL

Reply:
func (c *Client) FileCacheUploadAPI(req FileCacheUploadAPIReq) (res *FileCacheUploadAPIRes, err error) {
	url := c.BaseUrl
	Method := MethodPOST
	

is this okay?